docs/user-guide/resources-properties.md

```diff
@@ -0,0 +1,80 @@
+# Resource Properties
+------------------------------
+
+What a user can do on a resource (both maps and dashboards) depends on its role.
+
+* **Admin** users can manage accounts and can *view* and *edit* all the maps and dashboards, whoever created them
+
+* **Users** can view and edit only those maps and dashboards which they have created. If they are neither administrators nor owners, what they can do depends on the *Rights* assigned to the group they belong to
+
+* **Everyone** can view or edit the maps and dashboard according to the *rights* that have been assigned by the administrator or their owners
+
+In order to customize the properties of a resource, the admin or an user with permission can access the **Edit properties** window from the **Edit properties** button <img src="../img/button/edit-icon.jpg" class="ms-docbutton"/> in [Homepage](https://mapstore.geo-solutions.it/mapstore/#/) or from the **Save**/**Save as** button inside the resource viewer.
+
+<img src="../img/resource-properties/edit-properties.jpg" class="ms-docimage"  style="max-width:400px;"/>
+
+Through the **Edit properties** window the user can perform the following operations:
+
+* Add a **Thumbnail**
+
+* Add **Name** and **Description**
+
+* Add a **Permission** rule
+
+!!! warning
+    The name of a resource is the only mandatory field. Note that is not allowed to choose a name that has already been attributed to another resource.
+
+## Thumbnail
+
+It is possible to add an image as **thumbnail** dropping it or clicking inside the following box:
+
+<img src="../img/resource-properties/thumb.jpg" class="ms-docimage"/>
+
+!!! warning
+    The image to be added must not be larger than 500 kb and its best dimensions are 300x180 px.
+
+## Permission rule
+
+In the **Add a rule...** section you can set one ore more permission rules in order to allow everyone and/or specific groups of users to access the resource. Moreover it is possible to choose between two different ways with witch the selected group can approach the resource:
+
+* **View** the map and save a copy
+
+* **Edit** the map and re-save it
+
+In order to add a rule, the user can select the group and the access modality in the *Add a rule...* section. Once the rule is set, with the  **Add** button <img src="../img/button/add-rule-icon.jpg" class="ms-docbutton"/> it is possible to add it to the **Permissions Groups** list. <br>
```

Comment:
In order to add a rule, the user can select the group and set permissions inside the Add a rule... section